alacritty/src/config/mod.rs

```diff
@@ -154,7 +147,7 @@ pub fn reload_from(path: &PathBuf) -> Result<Config> {
 }
 
 fn read_config(path: &PathBuf) -> Result<Config> {
-    let mut contents = std::fs::read_to_string(path)?;
+    let mut contents = read_to_string(path)?;
```

Comment:
Functions that aren't in this file usually should have like one level of path to it so we know where things are coming from.

In this case it would be `fs::read_to_string`. That's usually the best middle-ground for functions and at least in Alacritty it should be the most common thing.